### DISCUSSION TITLE
Validation errors from laravel page load itself doesn't work in livewire components

### DISCUSSION THREAD

**Discussion by yungifez** [livewire/livewire#6546] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a form request and add some validation rules
2. Create a livewire component and place the livewire component into a blade file ( maybe a layout file )
3. Add the following to check for errors in both the livewire file and the blade file 
   ```blade 
       <div class="w-full" x-data="{open: true}" x-show="open" x-transition wire:ignore>
      @if ($errors->$errorBag->any())
          <div class=" bg-red-100 rounded-md dark:bg-red-600 dark:text-white text-red-900 border-red-300 border w-full">
              <div class="py-2 px-4 text-black dark:text-white flex justify-end">
                  <button type="button"  @click="open = false">
                      <i class="fa fa-x" aria-hidden="true"></i>
                  </button>
              </div>
              <ul class="pb-6 px-6">
                  <p class="font-bold text-lg"> Whoops! Something went wrong.</p>
                  @foreach ($errors->all() as $error)
                      <div class="pl-6">
                          <li class=" list-disc">{{ $error }}</li> 
                      </div>
                  @endforeach
              </ul>
          </div>
      @endif
  </div>
   ```
This should display all errors
4.  Create a form, submit the form ( without using livewire ) and make sure validation fails

It'll only display the first set of errors in the blade but wouldn't display the second in the livewire file

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by yungifez** [comment]
A possible fix is #6715

**Comment by lrljoe** [comment]
Please give a fuller example, include a basic example of how you're using the form in relation to the component, how the validation is being called etc.

Don't forget that different classes may be isolated from one another, and you may need to merge results together.

**Comment by LeeJump24** [comment]
Experiencing this issue as well.

**Reply by yungifez** [reply]
Livewire Core didn't accept my PR for this

So you could painfully set the error bag yourself while mounting
